pyproject.toml

```
@@ -0,0 +1,57 @@
+[build-system]
+requires = ["setuptools>=61.0.0", "setuptools_scm", "wheel>=0.36.1", "Cython>=0.29.28"]
+build-backend = "setuptools.build_meta"
```

As far as I can tell, `setuptools_scm` is not actually used by the project, and it builds without them. It probably be removed from these dependencies.

```suggestion
requires = ["setuptools>=61.0.0", "wheel>=0.36.1", "Cython>=0.29.28"]
build-backend = "setuptools.build_meta"
```

---

I don't see it used, either.